utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/wifi.lua

```diff
@@ -4,8 +4,8 @@ local iwinfo = require "iwinfo"
 local function scrape()
   local metric_wifi_network_quality = metric("wifi_network_quality","gauge")
   local metric_wifi_network_bitrate = metric("wifi_network_bitrate","gauge")
-  local metric_wifi_network_noise = metric("wifi_network_noise","gauge")
-  local metric_wifi_network_signal = metric("wifi_network_signal","gauge")
+  local metric_wifi_network_noise = metric("wifi_network_noise_db","gauge")
```

Comment:
`db` is a base unit I just made up, do you think that makes sense?

Reply:
https://github.com/prometheus/node_exporter/blob/859a825bb84bf4fb911fcba445d410965945f088/collector/wifi_linux.go#L115
I think correct unit would be dbm